cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -131,3 +148,19 @@ func RemovePrivacyLogTag(message string) string {
 
 	return message
 }
+
+// WorkflowStepIterator iterates over all workflow steps, including included
+// workflow steps, and calls cb callback function
+func WorkflowStepIterator(workflow *daisy.Workflow, cb func(step *daisy.Step)) {
+	if workflow == nil {
```

Comment:
How about making this a method directly on workflow?

Reply:
Moved to workflow.go

Reply:
Also renamed per Tony's suggestion and made it a member func